### PR TITLE
Expose progressRing Ellipse Diameter Propertie Feature Request

### LINKED ISSUE #1431: Expose Progress Ring Ellipse Diameter Properties
Currently it's set as width/8 which is not flexible for many practical use. It would be nice to expose some of the private properties so users can customize. Thanks.

### PR DESCRIPTION
**Pull request by HugzM**
fixes #1431  
I Change the visibility of EllispeDiameter setter on progressRing control.
It's now possible to set it throughxaml.

the ellipseDiameter is now completly independant from progress ring width.

It's my first contribution to this project and on opensource project in general.
I hope it's match with your expectations

### CODE REVIEW

**Comment by punker76** [comment]
@HugzM please make always a branch and after that a pr, thx

**Comment by thoemmi** [comment]
Instead of setting the default value for `EllipseDiameter` in the style in `ProgressRing.xaml`, you should  set it in the declaration of the dependency property, like this:

```C#
public static readonly DependencyProperty EllipseDiameterProperty = 
    DependencyProperty.Register("EllipseDiameter", typeof(double), typeof(ProgressRing), 
        new PropertyMetadata(7));
```

File: MahApps.Metro/Controls/ProgressRing.cs [hunk 1]
```diff
@@ -23,6 +23,8 @@ public class ProgressRing : Control
 
         public static readonly DependencyProperty EllipseDiameterProperty = DependencyProperty.Register("EllipseDiameter", typeof(double), typeof(ProgressRing), new PropertyMetadata(default(double)));
 
+
```
Comment (flagbug) [inline MahApps.Metro/Controls/ProgressRing.cs]: Where are these newlines coming from?

File: MahApps.Metro/Controls/ProgressRing.cs [hunk 2]
```diff
@@ -97,8 +99,8 @@ private static void BindableWidthCallback(DependencyObject dependencyObject, Dep
 
             var action = new Action(() =>
             {
-                ring.SetEllipseDiameter(
```
Comment (flagbug) [inline MahApps.Metro/Controls/ProgressRing.cs]: Just delete the code, don't comment it out

**Comment by HugzM** [comment]
@punker76 I created a new branch for this request but i can't edit this PR to change the target.

Should I create a new PR ? 
@thoemmi : you're right , i removed the default value and set it by the dependency property

**Pull request closed without merging**